test/support/api/outreach.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'underscore';
+
+Cypress.Commands.add('routeOutreachStatus', (mutator = _.identity) => {
+  cy.route({
+    url: '/api/outreach?*',
+    response() {
+      return mutator({
+        data: {
+          type: 'Outreach',
+          id: '11111',
+          attributes: {
+            phone_end: '1234',
+          },
+        },
```

Comment:
Worth it to move this to it's own fixture file?

Seemed overkill since it would only be one fixture, but not sure...